Makefile

```diff
@@ -37,7 +37,9 @@ lint-apply: lint-black-apply lint-isort-apply ## apply all the linter's suggesti
 
 .PHONY: test
 test:
-	python ./dev_scripts/pytest-wrapper.py -v --cov --ignore dev_scripts
+	#python ./dev_scripts/pytest-wrapper.py -v --cov --ignore dev_scripts
+	pytest -v --cov --ignore dev_scripts
```

Comment:
If the wrapper is not working out we should remove it. I have also needed to do so in the past since parallel tests fail with Qt.

Reply:
Yeah, definitely. I've marked this actually as a WIP commit, so that I can rebase on top of your own PR that deals with the same issue and ditch it.